app/views/administrate/application/_collection.html.erb

```diff
@@ -18,13 +18,13 @@ to display a collection of resources in an HTML table.
 [1]: http://www.rubydoc.info/gems/administrate/Administrate/Page/Table
 %>
 
-<table class="collection-data">
+<table class="collection-data" aria-labelledby="page-title">
```

Comment:
Question is, how does caption map to the accessibility API. I suspect it acts as the accessible name of the table.